### PR TITLE
Texts (fixing tests)

### PR DESCRIPTION
**Pull request by BrandonHowe**
New pull request, just fixed all the tests

### CODE REVIEW

File: packages/api/src/modules/users/router.ts [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    await next();
+});
+
+router.get("/forgotPassword/:key", async (ctx, next) => {
+    const { key } = ctx.params;
+
+    if (await keyValid(key)) {
+        ctx.status = 200;
+        ctx.redirect(
+            `${process.env.SERVER_URL}/#/loginregister/resetPassword/${key}`
```
Comment (vycdev) [inline packages/api/src/modules/users/router.ts]: Pretty sure that SERVER_URL is the url of the api and CORS_ORIGIN is the url of the front end.

File: packages/api/src/modules/users/router.ts [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+        );
+    } else {
+        ctx.status = 400;
+        ctx.redirect(`${process.env.SERVER_URL}/#/loginregister/invalidKey`);
```
Comment (vycdev) [inline packages/api/src/modules/users/router.ts]: Same, SERVER_URL is the url of the api

**Review by vycdev — CHANGES_REQUESTED** [review]

File: packages/api/src/modules/users/router.ts (hunk 1 above)
Comment (BrandonHowe) [reply packages/api/src/modules/users/router.ts]: Yeah, that seems about right. I mistakenly put CORS_ORIGIN there, forget about it. Plus, it's a monorepo

File: packages/api/src/modules/users/router.ts (hunk 2 above)
Comment (BrandonHowe) [reply packages/api/src/modules/users/router.ts]: Look above

**Pull request merged**